### PR TITLE
[BLOG] new post - call for myst templates

### PR DESCRIPTION
**Pull request by stevejpurves**
A blog post making a call for `myst` LaTeX/PDF template submissions that also help to announce the related functionality.

In terms of scheduling, this is probably better timed after a post announcing the JS tools -- so we can adjust the date here accordingly.

### CODE REVIEW

**Comment by stevejpurves** [comment]
This has been updated with a new possible publication date for next week! and with details of the upcoming webinar that @rowanc1 and I are hosting in Second week of March. This is a pretty straightforward post and we really like to get this out there to help raise awareness of what the myst cli can do and get some additional journal (and other) templates added to the collection.

@mmcky would you be able to give this a read over? I think if we have one review from someone else in the team then we can then probably post it!

**Comment by mmcky** [comment]
thanks for the ping @stevejpurves -- I will try take a look at it soon (and is on my list for Monday if I don't get to it before then).

File: docs/blog/2023-01-31-call-for-myst-templates.md [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+:width: 100%
+
+Use MyST Markdown to export to any template *LaTeX* template. To change the template you are using, just change the template option in frontmatter!
+```
+
++++ {"tags":[]}
+
+## MyST Markdown Template Repository
+
+We're excited to make a call for _LaTeX_ submissions to the new [MyST Template Repository](https://github.com/myst-templates). MyST templates allow any existing journal(Let’s not stop at journal papers — this equally applies to thesis and research report templates too!) _LaTeX_ template to be easily built directly from metadata and content in MyST Markdown documents.
```
Comment (mmcky) [inline docs/blog/2023-01-31-call-for-myst-templates.md:43]:
```suggestion
We're excited to make a call for _LaTeX_ submissions to the new [MyST Template Repository](https://github.com/myst-templates). MyST templates allow any existing journal's  _LaTeX_ template to be easily built directly from metadata and content in MyST Markdown documents. Let’s not stop at journal papers — this equally applies to thesis and research report templates too!
```

**Review by mmcky — COMMENTED** [review]
@stevejpurves thanks for putting this together. I had some very minor comments in the first `blog` post document. 

 I am not sure why there are two `blog` posts? Are these for two different workshops / dates?

**Comment by stevejpurves** [comment]
@mmcky my bad -- there should only be one -- it's a renaming mistake

**Comment by mmcky** [comment]
thanks @stevejpurves. 

This doesn't seem to show up in the preview build https://executablebookproject--929.org.readthedocs.build/en/929/blog/

I will try and checkout why later today.

File: docs/blog/2023-03-06-call-for-myst-templates.md [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+> Let’s remove the chore of formatting from the workload of scientists and researchers worldwide.
+
++++ {"tags":[]}
+
+```{figure} images/myst-export-to-any-template.png
+:name: seGB9FVdRJ
+:align: center
+:width: 100%
+
+Use MyST Markdown to export to any template *LaTeX* template. To change the template you are using, just change the template option in frontmatter!
```
Comment (rowanc1) [inline docs/blog/2023-03-06-call-for-myst-templates.md:36]:
```suggestion
Use MyST Markdown to export to any *LaTeX* template. To change the template you are using, just change the template option in frontmatter!
```

File: docs/blog/2023-03-06-call-for-myst-templates.md [hunk 3]
```diff
@@ -0,0 +1,178 @@
...
++++ {"tags":[]}
+
+> A single MyST template takes about an hour to build from an existing journal provided example… and can have a significant positive impact on time saved on all future submissions made via MyST.
+
++++ {"tags":[]}
+
+```{note}
+**Call for MyST Templates**
+
+With the [myst.tools](https://myst.tools) now launched (beta) 🚀 and publicly available, we are looking for *LaTeX* template submissions from the community.
```
Comment (rowanc1) [inline docs/blog/2023-03-06-call-for-myst-templates.md:66]:
```suggestion
With the [myst.tools](https://myst-tools.org) now launched (beta) 🚀 and publicly available, we are looking for *LaTeX* template submissions from the community.
```

File: docs/blog/2023-03-06-call-for-myst-templates.md [hunk 4]
```diff
@@ -0,0 +1,178 @@
...
+
+Contributing a *LaTeX* template is simple:
+
+1. Find an up-to-date *LaTeX* template for the venue or an example manuscript
+2. Follow the guide for [Creating a Template on the myst.tools site](https://myst.tools/docs/jtex/create-a-latex-template)
```
Comment (rowanc1) [inline docs/blog/2023-03-06-call-for-myst-templates.md:71]:
```suggestion
2. Follow the guide for [Creating a Template](https://myst-tools.org/docs/jtex/create-a-latex-template)
```

File: docs/blog/2023-03-06-call-for-myst-templates.md [hunk 5]
```diff
@@ -0,0 +1,178 @@
...
+3. Publish your template to GitHub and [add it to the index listing on myst-templates/templates](https://myst.tools/docs/jtex/contribute-a-template#list-your-template)
```
Comment (rowanc1) [inline docs/blog/2023-03-06-call-for-myst-templates.md:72]:
```suggestion
3. Publish your template to GitHub and [add it to the index listing on myst-templates/templates](https://myst-tools.org/docs/jtex/contribute-a-template#list-your-template)
```

File: docs/blog/2023-03-06-call-for-myst-templates.md [hunk 6]
```diff
@@ -0,0 +1,178 @@
...
+
+The template can stay on your GitHub account, making it easy for you to maintain and update. If this is likely to be a widely used template you might want to “donate” your contribution as a community-curated template, and transfer it to the `myst-templates` org — [read more about doing that here](https://myst.tools/docs/jtex/contribute-a-template#donate-your-template).
```
Comment (rowanc1) [inline docs/blog/2023-03-06-call-for-myst-templates.md:74]:
```suggestion
The template can stay on your GitHub account, making it easy for you to maintain and update. If this is likely to be a widely used template you might want to “donate” your contribution as a community-curated template, and transfer it to the `myst-templates` org — [read more about doing that here](https://myst-tools.org/docs/jtex/contribute-a-template#donate-your-template).
```

File: docs/blog/2023-03-06-call-for-myst-templates.md [hunk 7]
```diff
@@ -0,0 +1,178 @@
...
+
+```
+
++++ {"tags":[]}
+
+## How authors use MyST Templates
+
+Once a template is added to the repository it automatically becomes available to **the entire research community** via the open source [myst.tools](https://myst.tools) software ecosystem. Any MyST Markdown document of notebook can be exported to a template using the frontmatter ([learn more](https://myst.tools/docs/mystjs/creating-pdf-documents)).
```
Comment (rowanc1) [inline docs/blog/2023-03-06-call-for-myst-templates.md:82]:
```suggestion
Once a template is added to the repository it automatically becomes available to **the entire research community** via the open source [myst-tools.org](https://myst-tools.org) software ecosystem. Any MyST Markdown document of notebook can be exported to a template using the frontmatter ([learn more](https://myst-tools.org/docs/mystjs/quickstart-myst-documents)).
```

File: docs/blog/2023-03-06-call-for-myst-templates.md [hunk 8]
```diff
@@ -0,0 +1,178 @@
...
+
+```yaml
+---
+exports:
+  - format: pdf
+    template: lapreprint
+---
+```
+
+To find the templates available, use the `myst` [command line interface](https://myst.tools/docs/mystjs/quickstart) to search through all known templates:
```
Comment (rowanc1) [inline docs/blog/2023-03-06-call-for-myst-templates.md:92]:
```suggestion
To find the templates available, use the `myst` [command line interface](https://myst-tools.org/docs/mystjs/quickstart) to search through all known templates:
```

File: docs/blog/2023-03-06-call-for-myst-templates.md [hunk 9]
```diff
@@ -0,0 +1,178 @@
...
+```
+
+As you can see there is helpful and detailed information on the various parts and options that the template supports, making it easy to complete the document frontmatter to create the template.
+
+```{important}
+**Writing a paper Faster with MyST Markdown 📺**
+
+Looking to use the MyST Markdown tools for your writing?
+
+Get started easily by following the [quickstart tutorials](https://myst.tools/docs/mystjs/quickstart) or [watch our webinar](https://curvenote.com/blog/writing-a-scientific-paper-faster-myst-markdown).
```
Comment (rowanc1) [inline docs/blog/2023-03-06-call-for-myst-templates.md:129]:
```suggestion
Get started easily by following the [quickstart tutorials](https://myst-tools.org/docs/mystjs/quickstart) or [watch our webinar in the tutorial](https://myst-tools.org/docs/mystjs/quickstart-myst-documents).
```

File: docs/blog/2023-03-06-call-for-myst-templates.md [hunk 10]
```diff
@@ -0,0 +1,178 @@
...
+
++++ {"tags":[]}
+
+## Creating a MyST Template
+
+Ok, so having seen how rich the interaction with MyST Template is for authors, you may be thinking that building these templates might be quite onerous. This is not the case! If you have some _LaTeX_ experience and have the template and sample (e.g. content provided by a journal) the process is straightforward.
+
+There is a dedicated command line tool to help with the process — its called `jtex` — and it bends over backwards to help you create a MyST Template. Use it alongside the well-known `cookiecutter` tool to go from a clean slate to a local git repository that is ready to push.
+
+To get started for real, follow the [Create a Template](https://myst.tools/docs/jtex/create-a-latex-template) guide on [myst.tools](https://myst.tools), as it covers the whole process. To show you a little of what’s in store, the image below shows the output from `jtex check` part way through a template build.
```
Comment (rowanc1) [inline docs/blog/2023-03-06-call-for-myst-templates.md:141]:
```suggestion
To get started for real, follow the [Create a Template](https://myst-tools.org/docs/jtex/create-a-latex-template) guide on [MyST Tools](https://myst-tools.org), as it covers the whole process. To show you a little of what’s in store, the image below shows the output from `jtex check` part way through a template build.
```

File: docs/blog/2023-03-06-call-for-myst-templates.md [hunk 11]
```diff
@@ -0,0 +1,178 @@
...
+```
+
+```{tip} Sign up for the Upcoming MyST Templates Tutorial Webinar
+:class: dropdown
+
+Interested in porting your first LaTeX template to a MyST template?
+
+Join the upcoming webinar (15th March 2022 @ 4pm UTC) and get started (and maybe finished) building your template during the session.
+
+The webinar is hosted by Rowan & Steve from Curvenote, and will be a 2 hour walk-through at an easy pace of how to markup an existing LaTeX template or document for use with the [myst.tools](https://myst-tools.org) cli.
```
Comment (rowanc1) [inline docs/blog/2023-03-06-call-for-myst-templates.md:173]:
```suggestion
The webinar is hosted by Rowan & Steve from Curvenote, and will be a 2 hour walk-through at an easy pace of how to markup an existing LaTeX template or document for use with the [MyST Tools](https://myst-tools.org) cli.
```

File: docs/blog/2023-03-06-call-for-myst-templates.md [hunk 12]
```diff
@@ -0,0 +1,178 @@
...
+We are also seeing missing elements in the package manifest, which can be usually fixed automatically by typing `jtex check --fix`, `jtex` even scans all `tex`, `cls` and `sty` files you add to the `files` section of `template.yml` make sure that the packaging listing is correct, as it is used to ensure no duplicate packages are loaded later.
+
+```{important}
+**Contribute your first MyST template!**
+
+We’re on a mission to build the largest repository of open journal templates in the world. One that can be used to easily build papers from content in MyST Markdown and Juptyer Notebooks.
+
+Together let’s take the chore and re-work out of preparing a paper submission and make it possible to easily switch papers to alternate journal formats without wasting researchers’ time.
+
+[Make a start on your first MyST Template](https://myst.tools/docs/jtex) and submit it to the listing, or make a start and reach out to the [MyST templates community](https://github.com/myst-templates) in an issue to get help in getting it over the line.
```
Comment (rowanc1) [inline docs/blog/2023-03-06-call-for-myst-templates.md:162]:
```suggestion
[Make a start on your first MyST Template](https://myst-tools.org/docs/jtex) and submit it to the listing, or make a start and reach out to the [MyST templates community](https://github.com/myst-templates) in an issue to get help in getting it over the line.
```

**Review by rowanc1 — COMMENTED** [review]
A few changes to links post domain change!

**Comment by rowanc1** [comment]
Hoping that we can get this out early next week so that we have a few weeks to promote it before giving the webinar on the 15th. @mmcky let us know if you need any other help on this, I am not quite sure how to diagnose getting the blog to show up or get past the sphinx errors.

I think this is failing on the "Retrieving feature voting issue data..." Maybe it just needs a moment?!

**Comment by stevejpurves** [comment]
is iit a matter of someone with RTD access retriggering the build? as bad gateway does look like it could be a random network issue

**Comment by stevejpurves** [comment]
Great this went green ✅ -- all we need is an approving review!

**Pull request merged**